app/views/partials/paymentAllocations/PaymentAllocationsCreditAmount.scala.html

```diff
@@ -34,21 +35,36 @@
 
 @getLink = @{
     if (creditsRefundsRepayEnabled) {
-        link(id = Some("money-on-account-link"), link = creditAndRefundUrl, messageKey = "paymentAllocation.moneyOnAccount")
+        if(isAgent) {
```

Comment:
it would be nice to have patterns matching on tuple (creditsRefundsRepayEnabled, isAgent) here, but lets leave it for the next iteration